### ISSUE TITLE
Explain performance metrics, especially scoring, better

### ISSUE THREAD

**Issue by Meggin** [GoogleChrome/web.dev#685]
At the moment, it's tough for developers to understand why they have a particular performance score.

There are scoring guides for Lighthouse audits-- and you can try and dive into understanding how scoring works in performance through a range of docs, but it's really hard to follow.

We need to make this crystal clear.

Part of this clarity may come through improved UX in the Lighthouse report itself-- but we should also create this issue to make sure content covers the gaps-- and adds to making scoring in performance metrics crystal clear.

**Comment by jpmedley** [comment]
Is this a Lighthouse topic or a Performance APIs topic. If the later, let me know which APIs so that I can make sure MDN covers them.

**Comment by Meggin** [comment]
I have an idea for this. I'm going to organize audit content by categories, with metrics being a category type in performance audits. And the metrics docs will clearly explain what the metric is, how it is measured, and how it applies to scoring. It will also explain how to improve the metric in the context of load opportunities.

**Comment by Meggin** [comment]
Tracking this work in a new performance audit branch: https://github.com/GoogleChrome/web.dev/tree/performance-audits. Watch this space.

**Comment by Meggin** [comment]
Some work was done on this, but there's more work to do. We will be planning a new doc set for metrics, and will create new issue(s) to track,  so closing this now-- to at least mark the work done.
